### PR TITLE
Add e2e tests for select and selectattr operators

### PR DESCRIPTION
**Pull request by ttuffin**
Closes [AAP-9057](https://issues.redhat.com/browse/AAP-9057).

### CODE REVIEW

**Comment by ttuffin** [comment]
Tests are failing due to a dependency on #392. Once that is merged the tests should pass.

**Review by mkanoor — APPROVED** [review]

File: tests/e2e/files/rulebooks/operators/test_select_operator.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            has_moons: false
+            sub_type: "s-type"
+          - id: "testcase 05"
+            type: "comet"
+            name: "halleys"
+            radius: 11.0
+            orbital_period: 27284
+            has_moons: false
+
+  rules:
```
Comment (Alex-Izquierdo) [inline tests/e2e/files/rulebooks/operators/test_select_operator.yml:44]: We need at least one usecase with variable substitution, don't?

File: tests/e2e/files/rulebooks/operators/test_selectattr_operator.yml [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                  name: "sol"
+                  attributes:
+                    physical:
+                      age: 4600000000
+                      composition:
+                        gases:
+                          hydrogen: 73.46
+                          helium: 24.85
+
+  rules:
```
Comment (Alex-Izquierdo) [inline tests/e2e/files/rulebooks/operators/test_selectattr_operator.yml:108]: We need at least one usecase with variable substitution, don't?

Comment (ttuffin) [reply tests/e2e/files/rulebooks/operators/test_selectattr_operator.yml:110]: @Alex-Izquierdo thankyou for raising this, there should have been a note in each file pointing to the bug that needs to be resolved for this to work: https://issues.redhat.com/browse/AAP-9324

I've added a comment to both files.

**Review by Alex-Izquierdo — APPROVED** [review]

**Review by mkanoor — APPROVED** [review]

**Pull request merged**